docs/cleaning.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Data cleaning
+nav_order: 5
+---
+
+# Data cleaning
+
+Making datasets less noisy to improve quality of translation.
+
+## Regular pipeline
+
+
+Config setting:
+```
+  use-opuscleaner: false
+```
+
+### Dataset fixing
+
+Some datasets require fixes like detokenization. 
+Dataset and language specific fixes are implemented in [https://github.com/mozilla/firefox-translations-training/tree/main/pipeline/clean/fixes](https://github.com/mozilla/firefox-translations-training/tree/main/pipeline/clean/fixes).
+Naming convention: 
+- `<dataset_name>.sh` for parallel dataset cleaning
+- `<dataset_name>.<lang>.sh` for language specific cleaning of parallel or monolingual dataset
+- `/` in dataset name should be replaced with `_`
+
+### Cleaning scripts
+
+Make sure the language is present in [clean_parallel](https://github.com/mozilla/firefox-translations-training/tree/main/pipeline/clean/tools/clean_parallel.py#L19) script.
+
+
+### Bicleaner
```

Comment:
This is repeated in the training guide, it'd be better to have a single instance of it so we don't risk forgetting to update one or the other

Reply:
Good point, I added it for completeness. Replaced with a reference to the section in the training guide now. It's important to have it there I think.